GameServerCore/Packets/PacketDefinitions/Responses/AvatarInfoResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using GameServerCore.Enet;
```

Comment:
Is core still dependant on ENet?

Reply:
That's just folder with ClientInfo, not related to Enet. (why is this called Enet? I don't know)

Reply:
If it is so then it should be trivial ti rename the namespace.

Reply:
Changed.